libs/wire-api/src/Wire/API/Routes/Public/Galley/Conversation.hs

```diff
@@ -608,7 +609,7 @@ type ConversationAPI =
                :> QualifiedCapture "cnv" ConvId
                :> "subconversations"
                :> Capture "subconv" SubConvId
-               :> ReqBody '[JSON] DeleteSubConversationRequest
```

Comment:
Wouldn't this break older API versions?

Reply:
No, because the API is unchanged. Just the name of the type has changed.